Dockerfile

```
@@ -0,0 +1,30 @@
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with hishab.  If not, see <https://www.gnu.org/licenses/>.
+
+FROM golang:1.22.5-alpine AS builder
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -o /app/apiserver ./cmd/apiserver
+FROM alpine:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)